project/AkkaBuild.scala

```diff
@@ -87,7 +87,7 @@ object AkkaBuild {
     else Seq.empty,
     // should we be allowed to use artifacts from sonatype snapshots
     if (System.getProperty("akka.build.useSnapshotSonatypeResolver", "false").toBoolean)
-      resolvers ++= Resolver.sonatypeOssRepos("snapshots")
+      resolvers += Resolver.sonatypeCentralSnapshots
```

Comment:
This is probably a leftover from when we published snapshot to sonatype. I can't imagine that we need this any more. Remove it all together.